open-api.yaml

```diff
@@ -517,6 +517,71 @@ components:
         - type
         - link
       description: ''
+    key:
+      title: key
+      type: object
+      properties:
+        key:
+          type: string
+          description: The generated key. Generated by MeiliSearch.
+          example: d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4
+          readOnly: true
+        actions:
+          type: array
+          description: 'A list of indexes permitted for the key. ["*"] for all indexes.'
```

Comment:
I think this description and the description of `indexes` are switched.

Reply:
Thanks, @dichotommy! Good catch!